test/unit/index.html

```diff
@@ -4,19 +4,18 @@
     <meta charset="utf-8" />
     <title>Mocha Tests</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
-    <link rel="stylesheet" href="https://unpkg.com/mocha@2.5.3/mocha.css" />
+    <link rel="stylesheet" href="https://unpkg.com/mocha@9.2.2/mocha.css" />
   </head>
   <body>
     <div id="mocha"></div>
 
     <script src="https://unpkg.com/xhook@1.4.9/dist/xhook.js"></script>
-    <script src="https://unpkg.com/jquery@1.12.4/dist/jquery.js"></script>
+    <script src="https://unpkg.com/jquery@2.2.4/dist/jquery.js"></script>
```

Comment:
The jQuery version in [CONTRIBUTING.md](https://github.com/unravelin/ravelinjs/blob/3de07351cd98a7cd8aad9548dd315f89303be8af/CONTRIBUTING.md?plain=1#L111) can be bumped to v2.